docs/backends/impala.qmd

```diff
@@ -373,7 +373,7 @@ render_methods(
 ```
 
 To address a specific partition in any method that is partition
-specific, you can either use a dict with the partition key names and
+Specifically, you can either use a dict with the partition key names and
```

Comment:
This is correct as it was, but perhaps changing `that is partition specific` to `that is partition-specific` would make it more clear?